### DISCUSSION TITLE
Y-AXIS ROTARY ROLLER - SETUPS CAUSING PROBLEMS

### DISCUSSION THREAD

**Discussion by preissasaf** [arkypita/LaserGRBL#1675] [Q&A]
Hi,
I've installed and assembled my ORTUR OLM2 PRO S2 and used LaserGRBL and have done some engraving. Things went well till I installed the YRR2.0 Y-Axis rotary and configured it as shown on the brochure, meaning, using GRBL to write and change the "HOME CYCLE ENABLE" parameter to "0" (zero).
Since then, everything went wrong.
The laser goes way beyond its limits, doing nothing, doesn't read right the image dimensions and so on.
It's also affect the way the machine acts on other software interface, like LightBurn.
PLS FIND ATTACHED A PHOTO OF THIS BROCHURE.
Please HELP ASAP!!! 
Thanks,
Asaf
![YRR2 configuration on LaserGRBL](https://user-images.githubusercontent.com/95986780/146135128-b375e1d6-94aa-46c3-8ec1-23f070105145.jpg)

**Comment by gmmanonymus111** [comment]
What do you mean by
> Since then, everything went wrong.

?
Changing a wire (more precisely a stepper motor) shouldn't cause such problems.
The problem with crashing may be becuase you accidentally enabled homing without limit switches.

Could you please post your `$$` GRBL settings?

**Reply by Hostaler** [reply]
Cuál es el parámetro de retorno al inicio?

**Comment by Tribo-2dot0** [comment]
Quando você voltar a máquina para o modo normal, ou seja, remover o rotativo e religar o eixo Y da máquina, você deve voltar o valor de "Homing Cycle Enable" para o valor original ($22=1) refazendo os mesmos passos das instruções.
Baixe aqui os botões do rotativo Ortur para que não precise ficar fazendo todo esse processo para a troca:
[Botoes Rotativo Ortur.zip](https://github.com/user-attachments/files/16836872/Botoes.Rotativo.Ortur.zip)
